.gitignore

```diff
@@ -3,3 +3,4 @@ target/
 
 
 
+docs/shared_library?libscryer_prolog.h
```

Comment:
```suggestion
docs/shared_library/libscryer_prolog.h
```

Reply:
I think it was discussed somewhere to add the header to the repo so this should be removed from .gitignore